.idea/compiler.xml

```diff
@@ -7,14 +7,17 @@
         <sourceTestOutputDir name="target/generated-test-sources/test-annotations" />
         <outputRelativeToContentRoot value="true" />
         <module name="vocab-trainer" />
+        <module name="ASD-Afternoon-1" />
```

Comment:
Please, do not commit this file. This change is probably only connected to importing a project from a folder named "ASD-Afternoon-1" - the repository's name.